### ISSUE TITLE
Crash on iOS

### ISSUE THREAD

**Issue by vshymanskyy** [jacobrosenthal/react-native-ble#56]
```
react-native init TestApp
cd TestApp
npm install --save react-native-ble
rnpm link
react-native run-ios
```
The project builds fine, but crashes upon startap in RNBLE module.
Even without js actually using it.
Android works fine!

**Comment by vshymanskyy** [comment]
@jacobrosenthal is there any chance the library will be updated soon?
it looks like there's a bunch of PRs that could solve this (and other) issues.

**Comment by jacobrosenthal** [comment]
I think it’s safe to say I’m disconnected from th project at this point. If
there’s a fork users like I’ll hand to that or add some people here

**Comment by lucasltv** [comment]
@vshymanskyy did you fix the problem?

**Comment by vshymanskyy** [comment]
@lucasltv no, switched to https://github.com/innoveit/react-native-ble-manager for now
